Checkstyle errors with non-English locale and Java >= 9

# What steps will reproduce the problem?
`mvn clean verify`

# What error message or stack trace does EqualsVerifier give?
```
[INFO] --- maven-checkstyle-plugin:3.0.0:check (run-checkstyle) @ equalsverifier ---
[INFO] Début de la vérification...
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\integration\extended_contract\ExternalApiClassesTest.java:10: 'java.math.BigDecimal' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\internal\reflection\annotations\AnnotationCacheBuilderTest.java:20: 'java.util.Collections' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\testhelpers\annotations\DefaultNonnullCustom.java:4: 'java.lang.annotation.Documented' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\testhelpers\annotations\DefaultNonnullInapplicable.java:5: 'java.lang.annotation.Documented' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\testhelpers\annotations\DefaultNonnullJavax.java:6: 'java.lang.annotation.Documented' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\testhelpers\annotations\javax\annotation\Nonnull.java:4: 'java.lang.annotation.Retention' devrait être séparé des imports précédents. [ImportOrder]
[ERROR] C:\Users\xxx\GIT_PERSO\equalsverifier\src\test\java\nl\jqno\equalsverifier\testhelpers\ConditionalCompiler.java:5: 'java.io.Closeable' devrait être séparé des imports précédents. [ImportOrder]
Vérification terminée.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 54.647 s
[INFO] Finished at: 2018-10-08T21:00:31+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check (run-checkstyle) on project equalsverifier: Failed during checkstyle execution: There are 7 errors reported by Checkstyle 8.12 with project/checkstyle-config.xml ruleset. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

# What did you expect?
Build success

# Which version of EqualsVerifier are you using?
master

# Please provide any additional information below.
```
MINGW64 ~/GIT_PERSO/equalsverifier (master)
$ java -version
java version "11" 2018-09-25
Java(TM) SE Runtime Environment 18.9 (build 11+28)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11+28, mixed mode)

MINGW64 ~/GIT_PERSO/equalsverifier (master)
$ mvn -version
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T20:33:14+02:00)
Maven home: C:\opt\apache-maven-3.5.4
Java version: 11, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-11
Default locale: fr_FR, platform encoding: Cp1252
OS name: "windows 7", version: "6.1", arch: "amd64", family: "windows"
```

--- a/project/suppressions.xml
+++ b/project/suppressions.xml
@@ -4,5 +4,5 @@
     "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
-    <suppress checks="ImportOrder" message="'java.* should be separated from previous imports"/>
+    <suppress checks="ImportOrder" message="'java.*"/>
 </suppressions>

